dev-requirements.txt

```diff
@@ -18,4 +18,6 @@ click
 scikit-build
 setuptools_scm
 pytest-snapshot
-flake8>=4.0.0
\ No newline at end of file
+flake8>=4.0.0
+cmake-format
```

Comment:
Suggest we add the lower bound `>= 0.6.0`.

Reply:
Ehh - I was about to address this, but then it was merged. I'll leave it.

Reply:
No worries, just me being overly pedantic :)